src/pages/nft/claim/index.vue

```diff
@@ -1,81 +1,76 @@
-      :login-button-label="$t('nft_claim_login_in_button')"
-      :is-stick-to-bottom-at-mobile="true"
-    >
-      <template #prepend>
-        <NFTWidgetBaseCard class="flex justify-center items-center max-w-[400px] mb-[16px]">
-          <NuxtLink
-            :to="localeLocation({ name: 'nft-class-classId', params: { classId } })"
-            target="_blank"
-          >
-            <NFTWidgetContentPreview
-              :class="[
-                'transition-shadow',
-                'cursor-pointer',
-                'min-h-[300px]',
-                'w-full',
-              ]"
-              :title="NFTName"
-              :description="NFTDescription"
-              :img-src="NFTImageUrl"
-              @click="handleClickViewDetails"
-            />
-          </NuxtLink>
-        </NFTWidgetBaseCard>
-      </template>
-      <Label class="mb-[16px]" :text="text" align="center" />
-      <template v-if="state === 'INITIAL'">
+    <div class="relative flex flex-col justify-center items-center w-full max-w-[962px] mx-auto mt-[4px] px-[1rem] py-[1.5rem]">
+      <NFTWidgetBaseCard class="flex justify-center items-center max-w-[400px] mb-[16px]">
+        <NuxtLink :to="localeLocation({ name: 'nft-class-classId', params: { classId } })" target="_blank">
+          <NFTWidgetContentPreview :class="[
+            'transition-shadow',
+            'cursor-pointer',
+            'min-h-[300px]',
+            'w-full',
+          ]" :title="NFTName" :description="NFTDescription" :img-src="NFTImageUrl" @click="handleClickViewDetails" />
+        </NuxtLink>
+      </NFTWidgetBaseCard>
+      <template v-if="!claimingAddress">
         <div class="flex flex-col justify-center w-full max-w-[680px] px-[12px] mt-[-12px]">
-          <Label
-            preset="p6"
-            class="text-medium-gray mt-[12px] mb-[6px]"
-            :text="$t('nft_collect_modal_leave_message')"
-          />
-          <div class="flex w-full py-[10px] px-[16px] gap-[12px] bg-shade-gray rounded-[12px]">
-            <IconMessage class="text-dark-gray" />
-            <input
-              v-model="collectorMessage"
-              type="input"
-              class="w-full bg-transparent border-0 focus-visible:outline-none"
-              :placeholder="$t('nft_collect_modal_leave_message_to_name', { name: creatorDisplayName })"
-              @input.once="onInputCollectorMessage"
-            >
-          </div>
-          <ButtonV2
-            class="self-center mt-[24px]"
-            :text="$t('nft_claim_claim')"
-            preset="secondary"
-            @click="claim"
-          />
+          <template v-if="walletIsLoggingIn">
+            <ProgressIndicator />
+            <Label class="text-medium-gray w-full mt-[4px]" align="center" preset="p6"
+              :text="$t('auth_required_view_hint_label_loading')" />
+          </template>
+          <template v-else>
+            <div class="flex flex-col justify-center w-full px-[12px] mt-[12px]">
```

Comment:
I don't think is it necessary to use this `div` as we have similar classes in parent `div` in L15, just moved the `mt-[12px]` to the parent

Parent
<img width="732" alt="image" src="https://github.com/likecoin/liker-land/assets/17264716/6a2b202f-a7d3-49fe-a94a-a727a5ad71ac">

This `div`
<img width="710" alt="image" src="https://github.com/likecoin/liker-land/assets/17264716/75bcbafa-6d27-4dbc-8d0b-18eb3f19d66a">

Reply:
this div also breaks the claim NFT button to next line though